src/main/scala/org/antipathy/scoozie/action/prepare/Prepare.scala

```diff
@@ -1,32 +1,35 @@
 package org.antipathy.scoozie.action.prepare
 
-import scala.xml.Elem
-import scala.collection.immutable.Map
 import org.antipathy.scoozie.action.filesystem.{Delete, MakeDir}
 import org.antipathy.scoozie.xml.XmlSerializable
+
 import scala.collection.immutable._
+import scala.xml.Elem
 
 /**
   * Ooize actions prepare definition
   * @param actions the prepare actions
   */
 case class Prepare(actions: Seq[PrepareFSAction]) extends XmlSerializable {
+  import org.antipathy.scoozie.configuration.ActionProperties
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)